source/boards/work-with-cards.rst

```diff
@@ -20,5 +20,97 @@ Work with Cards
 
-The default name for a new property is **New Property**. Ensure that you name your property as it's added to the filter list at the top of the board and having all your filters named **New Property** makes it difficult to work out what you're filtering. You can rename a property field by opening up a card and selecting **New Property**. Enter the new name in the field provided. The change is saved immediately and applied across all cards.
+The default name for a new property is **New Property**. Ensure that you name your property as it's added to the filter list at the top of the board and having all your filters named **New Property** makes it difficult to work out what you're filtering. To rename a property field, open up a card and select **New Property**. Enter the new name in the field provided. The change is saved immediately and applied across all cards.
 
+Mention people
+--------------
+
+You can include a team member on a card by mentioning them on a card <https://docs.mattermost.com/messaging/mentioning-teammates.html>__ the same way you would in Channels. The team member you mention will receive a Direct Message notification from the Boards bot with a link to the card you mentioned them on. To mention multiple team members, separate each name with a comma.
+
+Share card previews
+-------------------
+
+When you share a link to a card within Mattermost, the card details are automatically displayed in a preview. This preview highlights what the card is about at a glance without having to navigate to it.
+
+Customize card properties
+-------------------------
+
+Cards can contain different data fields depending on the purpose of the board. For example, in a Roadmap board, cards include a Type field where you can add categories such as Bug, Triage, etc. In a Task board, cards include Status and Priority fields instead.
+
+Add and manage properties
+-------------------------
+
+To create a new a new property field open a card and select **Add a property**. Then select the type of property from the drop-down menu. The property type specifies the type of data you plan to capture within that field. When you create new card properties, they are added to all new and all existing cards on the current board.
+
+Properties are automatically added to the board filter list at the top of the page, so ensure you customize all property names to make it easy to filter your board by specific properties later. You can rename a property at any time by opening a card, then selecting **New Property**. Your changes are saved immediately, and applied across all cards on the current board.
```

Comment:
You can rename a property at any time by opening a card, then clicking on the **property name** to open an editable field.